js/ui/iconGrid.js

```diff
@@ -57,6 +57,7 @@ const EditableLabel = new Lang.Class({
         this.parent(params);
 
         this.clutter_text.editable = false;
+        this.clutter_text.selectable = false;
```

Comment:
Drive-by comment — you could even do:

```
this.clutter_text.bind_property('editable', this.clutter_text, 'selectable', GObject.BindingFlags.BIDIRECTIONAL | GObject.BindingFlags.SYNC_CREATE);
```

inside the instance initialization, so you would not need the rest of the patch.